src/components/DataEntryModal/DataEntryModal.jsx

```diff
@@ -135,6 +167,15 @@ const DataEntryModal = ({
   // };
   // control.getFieldState('pounds').setValue(pounds);
 
+  const updateTags = async id => await getImagesByEventID(id);
+  useEffect(() => {
+    updateTags(id).then((data) => setTags(data));
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'id'. Either include it or remove the dependency array.